pkg/util/kubeconfig.go

```diff
@@ -61,7 +61,7 @@ func GetWDSKubeconfig(logger logr.Logger, wdsName, wdsLabel string) (*rest.Confi
 			return nil, "", err
 		}
 	}
-	logger.Info("using label", "key", label.Key, "value", label.Value)
+	logger.Info("Using label", "wdsName", wdsName, "keyJustKidding", label.Key, "value", label.Value)
```

Comment:
Why the "keyJustKidding" in the log message? It seems confusing. Also, it says "using label" but then it prints the wdsName and value. I would change the log pattern this way:

logger.Info("Using label",  "name", label.Key, "value", label.Value,"wdsName",wdsName)

Reply:
The "JustKidding" part is an interim marker that this needs to be fixed as part of fixing #2133 , and a warning to anyone who reads such a log message before the fix.

Reply:
The force-push to 9df116506 changes the order as suggested here.

Reply:
The force-push to 359b169b7 makes this moot.